### ISSUE TITLE
a significant difference between the predictions and corresponding labels in the generated preds_on_lnp_hela_with_feat.csv file

### ISSUE THREAD

**Issue by wklix** [bowang-lab/AGILE#5]
Thank you for your modifications to the infer_viz.py last month. By running infer_viz.py, I noticed a significant difference between the predictions and corresponding labels in the generated preds_on_lnp_hela_with_feat.csv file. Does this indicate that the fine-tuned model performs poorly? Additionally, I would like to know how to run your code to achieve the selection of the final candidate set of 15 LNPs.

**Comment by rexxxx1234** [comment]
Hi @wklix, thanks for your question! Could you please elaborate a bit more on the significant difference between the predictions and corresponding labels? The performance of the fine-tuned model on the fine-tuning set is detailed in our manuscript (Fig 2g). 

For selecting the top 15 LNPs,  I can provide a notebook with the selection code.

**Comment by wklix** [comment]
Thank you for your reply!  The significant difference between the predictions and corresponding labels is in the attachment of the mailbox, where you can see the big difference between lable and pred. In addition,

thank you very much for what you can provide a notebook with the selection code. May I ask if you sent this notebook by email？




At 2023-08-08 22:19:56, "Rex Ma" ***@***.***> wrote:

Hi @wklix, thanks for your question! Could you please elaborate a bit more on the significant difference between the predictions and corresponding labels? The performance of the fine-tuned model on the fine-tuning set is detailed in our manuscript (Fig 2g).

For selecting the top 15 LNPs, I can provide a notebook with the selection code.

**Comment by rexxxx1234** [comment]
Hi @wklix, I wasn't able to find the attachment; could you send it to my email: rexma1102@gmail.com?

**Comment by 9udrun** [comment]
Hi, I believe I am running into the same problems when I do the finetuning and inference, for example the predicted expression with the H9 lipid is only ranked 600-800th. I would love to see if doing the ensemble prediction and candidate selection like in the paper would give me more similar results. 
Could you send me a copy of the selection code notebook? My email is lallileyni@gmail.com

Thank you for publishing the code along with the paper, your work is very interesting!
